modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/milker.dm

```diff
@@ -0,0 +1,319 @@
+/// Volume 'light' mode extracts per action
+#define VOLUME_LIGHT 1
+/// Volume 'strong' mode extracts per action
+#define VOLUME_STRONG 5
+/// 'light' mode
+#define HANDLE_LIGHT 1
+/// 'strong' mode
+#define HANDLE_STRONG 2
+/// The lowest numbered setting currently available (so we dont need yet another list)
+#define HANDLE_MODE_LOWEST 1
+/// The highest numbered setting currently available
+#define HANDLE_MODE_HIGHEST 2
+/// How much pleasure is gained through 'light' mode
+#define PLEASURE_LIGHT 1
+/// How much pleasure is gained through 'strong' mode
+#define PLEASURE_STRONG 3
+/// How much arousal is gained through 'light' mode
+#define AROUSAL_LIGHT 1
+/// How much arousal is gained through 'strong' mode
+#define AROUSAL_STRONG 3
+/// How much pain is gained through 'light' mode
+#define PAIN_LIGHT 3
+
+#define PAIN_STRONG 10 // How much pain is gained through 'strong' mode// Self-(and otherwise) Expression (of mammary organs)
```

Comment:
```suggestion
/// How much pain is gained through 'strong' mode
#define PAIN_STRONG 10

// Self-(and otherwise) Expression (of mammary organs)
```